tests/ui/landing_page_test.py

```diff
@@ -24,6 +24,12 @@ def setUp(self):
   def tearDown(self):
     """Teardown for test methods."""
     landing_page = LandingPage(self.driver)
+    try:
+      user_details_button = landing_page.get_element(
+          LandingPage.USER_DETAILS_BUTTON)
+      user_details_button.click()
+    except:
+      self.driver.get(self.args.server_url + flask.url_for('landing'))
```

Comment:
No, if the page gets reloaded then the details dialog won't be open at all so there's no need to close it.